interface/patient_tracker/patient_tracker.php

```diff
@@ -768,6 +768,7 @@ function openNewTopWindow(newpid,newencounterid) {
         $.datetimepicker.setLocale('<?= $DateLocale;?>');
     });
     $("#form_apptstatus").addClass('form-control input-sm');
+    $('#form_apptstatus').removeAttr('style');
```

Comment:
This is/was in another PR

Reply:
They were, but there are no conflicts as such.  They will mash together.

Reply:
He might not have been able to test with broken code...you know?